streaming/base/shared/prefix.py

```diff
@@ -184,29 +210,37 @@ def get_shm_prefix(streams_local: list[str],
     # Check my locals for overlap.
     _check_self(streams_local)
 
+    prefix_int = max([
```

Comment:
is there a case where the found prefix ints would be different for the different shm names? is this the duplicate shmem stuff / not getting cleaned thing you mention in the description?

Reply:
> is there a case where the found prefix ints would be different for the different shm names?

I strongly doubt that this will ever happen.